include/task.h

```diff
@@ -1,6 +1,10 @@
 #ifndef GUARD_TASK_H
 #define GUARD_TASK_H
 
+// general task defines
+#define TASK gTasks[taskId]
+#define FUNC func
```

Comment:
The comment was referring to `FUNC`. I never opposed a `TASK` define. I was the first to use it.

That warning would happen if you are putting the defines in other headers, which is misuse. The question was only ever whether it would go in task.h or each .c file that needed it.